src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class IsoscelesTrapezoid extends Figure {
```

Comment:
The class `IsoscelesTrapezoid` should not start with an empty line. Please remove the empty line at the beginning of the class definition.